quickstarts/iFrame.mdx

```diff
@@ -0,0 +1,297 @@
+---
+title: 'iFrame'
+description: ''
```

Comment:
Please add a description. E.g `Learn how to use Novu to receive In-App notifications with iFrame embed.`

Reply:
Done.